### DISCUSSION TITLE
Custom Button Entity

### DISCUSSION THREAD

**Discussion by 1RandomDev** [zachowj/node-red-contrib-home-assistant-websocket#537] [Feature Request]
Can we pls have custom button entities?

**Comment by frenck** [comment]
Home Assistant Core 2022.2.0 will have `input_button` / button helpers.

**Comment by zachowj** [comment]
Duplicate https://github.com/zachowj/node-red-contrib-home-assistant-websocket/discussions/530

**Reply by zachowj** [reply]
Added in v0.41.0
